Finjinon/PhotoCaptureViewController.swift

```diff
@@ -56,6 +56,11 @@ open class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLayo
     fileprivate let buttonMargin: CGFloat = 12
     fileprivate var orientation: UIDeviceOrientation = .portrait
 
+    private var viewFrame: CGRect!
```

Comment:
I know this class already implicitly unwraps a lot of things, but I'd prefer if we try to avoid this as much as possible going forward. Couldn't we just give these the default value `.zero` ?

Reply:
👍